### PR TITLE
Fix splash screen

### PR DESCRIPTION
**Pull request by antonmolodykh**

### CODE REVIEW

File: ansible/roles/splashscreen/files/screenly.plymouth [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Plymouth Theme]
+Name=Screenly
+Description=Screenly Splash
```
Comment (vpetersson) [inline ansible/roles/splashscreen/files/screenly.plymouth]: Screenly Splash Screen

**Comment by vpetersson** [comment]
@antonmolodykh We need to have the 'uninstall' part for the older versions such that they don't end up with both splashscreen modes.

**Comment by antonmolodykh** [comment]
@vpetersson added

File: ansible/roles/splashscreen/tasks/main.yml [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
 
 - name: Copies in rc script
   copy:
     src: asplashscreen
     dest: /etc/init.d/asplashscreen
     mode: 0755
+  when: ansible_distribution_major_version|int <= 7
 
 - name: Enables asplashscreen
   command: update-rc.d asplashscreen defaults
```
Comment (vpetersson) [inline ansible/roles/splashscreen/tasks/main.yml]: Don't you need an inverse of this too?

Comment (antonmolodykh) [reply ansible/roles/splashscreen/tasks/main.yml]: @vpetersson yes, you're right. added.

**Comment by vpetersson** [comment]
LGTM.

**Pull request merged**